packages/td-bank-integration/src/td-online-mart-payment-strategy.ts

```diff
@@ -112,13 +119,28 @@ export default class TDOnlineMartPaymentStrategy implements PaymentStrategy {
         return {
             methodId,
             paymentData: {
+                ...commonPaymentData,
                 nonce: paymentToken,
                 shouldSaveInstrument,
-                shouldSetAsDefaultInstrument,
             },
         };
     }
 
+    /* eslint-disable @typescript-eslint/naming-convention */
+    private getTDBrowserInfo(): TdOnlineMartBrowserInfo {
+        const { time_zone_offset, ...defaultBrowserInfo } = getBrowserInfo();
+
+        return {
+            ...defaultBrowserInfo,
+            accept_header:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3',
+            javascript_enabled: true,
+            time_zone: time_zone_offset,
+            user_agent: window.navigator.userAgent,
```

Comment:
Sending this in the JSON payload seems a bit odd to me:
- `User-Agent` and `Accept` should already be sent as header with the request
- Forcing `Accept` to a static value is probably not desired for the 3DS check
- javascript_enabled can probably be assumed to always be `true` in the backend, since the checkout process requires it

Reply:
Hi @TomA-R 
Thanks for info about headers. The main idea was to keep all needed data in one place, but if you think that better to collect some data on BE side I removed fields for `Accept`, `User_Agent` and `js-enabled`.

Reply:
Thanks @PavlenkoM, best to collect that from the actual requests 👍